readme.md

```diff
@@ -183,7 +183,24 @@ import {TailwindProvider} from 'tailwind-rn';
 import utilities from './tailwind.json';
 
 const App = () => (
-	<TailwindProvider utilities={utilities}>
+	<TailwindProvider utilities={utilities} theme="dark">
```

Comment:
This example isn't related to the introduction of the new prop, could you undo the change here?